pages/nft/url/index.vue

```diff
@@ -311,6 +353,12 @@ export default class FetchIndex extends Vue {
     if (this.url) {
       this.onInputURL(this.url);
     }
+
+    this.isAllowed = await this.checkIsWhitelisted();
+    if (!this.isAllowed) {
+      logTrackerEvent(this, 'IscnMintNFT', 'CreateNFTError', ErrorType.USER_NOT_WHITELISTED, 1);
+    }
+    this.isReady = true
```

Comment:
Maybe a big `try catch finally` to set `isAllowed` to `true` in case of error, and `isReady` to `true` in all cases